plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotClient.java

```diff
@@ -155,6 +156,7 @@ public PinotClient(
         this.brokerResponseCodec = requireNonNull(brokerResponseCodec, "brokerResponseCodec is null");
         this.pinotHostMapper = requireNonNull(pinotHostMapper, "pinotHostMapper is null");
         this.scheme = config.isTlsEnabled() ? "https" : "http";
```

Comment:
somewhere above this line we should `requireNonNull(pinotConfig, "pinotConfig is null")`